examples/getstarted/extensions/documentation/documentation/1.0.0/full_documentation.json

```diff
@@ -14,7 +14,7 @@
       "name": "Apache 2.0",
       "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "x-generation-date": "05/22/2019 6:18:29 PM"
+    "x-generation-date": "06/18/2019 5:15:23 PM"
```

Comment:
I really think we should add all the related documentation files into the .gitignore. What do you think?

Reply:
Sure I'll do another PR to remove them at once and add them to the gitignore